src/main/java/com/bullhornsdk/data/model/enums/BullhornEntityInfo.java

```diff
@@ -456,6 +460,7 @@ public enum BullhornEntityInfo {
     PERSON_CUSTOM_OBJECT_INSTANCE_23("PersonCustomObjectInstance23", PersonCustomObjectInstance23.class, PersonCustomObjectInstance23Wrapper.class, PersonCustomObjectInstance23ListWrapper.class, "PersonCustomObject23EditHistory", "PersonCustomObjectEditHistoryFieldChange"),
     PERSON_CUSTOM_OBJECT_INSTANCE_24("PersonCustomObjectInstance24", PersonCustomObjectInstance24.class, PersonCustomObjectInstance24Wrapper.class, PersonCustomObjectInstance24ListWrapper.class, "PersonCustomObject24EditHistory", "PersonCustomObjectEditHistoryFieldChange"),
     PERSON_CUSTOM_OBJECT_INSTANCE_25("PersonCustomObjectInstance25", PersonCustomObjectInstance25.class, PersonCustomObjectInstance25Wrapper.class, PersonCustomObjectInstance25ListWrapper.class, "PersonCustomObject25EditHistory", "PersonCustomObjectEditHistoryFieldChange"),
+    PERSON_CUSTOM_OBJECT_INSTANCE_29("PersonCustomObjectInstance29", PersonCustomObjectInstance29.class, PersonCustomObjectInstance29Wrapper.class, PersonCustomObjectInstance29ListWrapper.class, "PersonCustomObject29EditHistory", "PersonCustomObjectEditHistoryFieldChange"),
```

Comment:
Can we include PERSON_CUSTOM_OBJECT_INSTANCE_26 to PERSON_CUSTOM_OBJECT_INSTANCE_28 and PERSON_CUSTOM_OBJECT_INSTANCE_30?